src/App.jsx

```diff
@@ -37,56 +38,27 @@ function App() {
       if (response.data.count > 0) {
```

Comment:
You don't need an if statement here. Unless you have you have a specific message to display if it comes back with 0 results.